lib/index.js

```diff
@@ -0,0 +1,89 @@
+/*!
+ * Copyright (c) 2017 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const bedrock = require('bedrock');
+const brLedger = require('bedrock-ledger');
+const equihashSigs = require('equihash-signature');
+const validate = require('bedrock-validation').validate;
+const BedrockError = bedrock.util.BedrockError;
+
+require('./config');
+
+bedrock.events.on('bedrock.start', () => brLedger.use('EquihashValidator2017', {
+  type: 'validator',
+  api: api
+}));
+
+const api = {};
+// NOTE: only exported for tests
```

Comment:
Right, we need to decide exactly what we're trying to demonstrate in the readme.  Do we want to show how validators are used by consensus API's?  Right now, it shows how they can be used in a test.